src/swap/defi/thorchainDa.ts

```diff
@@ -86,17 +86,18 @@ const asThorSwapRoute = asObject({
   providers: asArray(asString),
   calldata: asUnknown,
   expectedOutput: asString,
-  expectedOutputMaxSlippage: asString,
-  expectedOutputUSD: asString,
-  expectedOutputMaxSlippageUSD: asString,
+  // expectedOutputMaxSlippage: asString,
+  // expectedOutputUSD: asString,
+  // expectedOutputMaxSlippageUSD: asString,
```

Comment:
They are no longer used. `expectedOutputMaxSlippage` was the only one used and we don't care about the max slippage anymore with variable quotes since we just quote the user the expected amount.

Note that convention is to comment out unused cleaner params so we could re-enable them in the future if needed.